task_8/RinaDish/src/main/java/com/hillel/task2/Shower.java

```diff
@@ -0,0 +1,39 @@
+package com.hillel.task2;
+
+public class Shower {
+
+    public void setColdWater(boolean coldWater) {
+        this.coldWater = coldWater;
+    }
+
+    public void setHotWater(boolean hotWater) {
+        this.hotWater = hotWater;
+    }
+
+    private boolean coldWater;
+    private boolean hotWater;
+    private String bathe;
+
+    String bathe() {
+        if ((coldWater == false) && (hotWater == false)) {
```

Comment:
You can do this like "if (coldWater && hotWater)". Others below apply with the same rule